spacy_llm/tasks/templates/raw.v1.jinja

```diff
@@ -0,0 +1,20 @@
+Read the instructions provided after "Text:" and reply after "Reply:".
+{# whitespace #}
+{%- if prompt_examples -%}
+Below are some examples (only use these as a guide):
+{# whitespace #}
+{%- for example in prompt_examples -%}
+{# whitespace #}
+Text:
+{{ example.text }}
+Reply:
+{{ example.reply }}
+{# whitespace #}
+{%- endfor -%}
+{# whitespace #}
+{%- endif -%}
+{# whitespace #}
+Now follows the text you should read and reply to.
+Text:
+{{ text }}
```

Comment:
If there are no examples, wouldn't it make more sense to just push the `{{text}}` out without any further instructions?

Reply:
It would also be more "true" to the promise of just sending the doc's content...

Reply:
Removed everything except for "Text:" and "Reply:". Do you thing we should toss these too?